src/pages/code-for-good.js

```diff
@@ -183,6 +181,13 @@ const StyledDashboardCaption = styled(P3)`
 
 const DashboardCaption = StyledDashboardCaption.withComponent('figcaption');
 
+const HeaderActionsContainer = styled('div')`
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
Using `flex-wrap` allows us to send the link to the next line instead of wrapping (nearly the same as using white-space: nowrap), but we also get more control over alignment with flexbox (such as centering elements so text aligns).